### PR TITLE
feat: Add new questions to security quiz

### PR DESCRIPTION
**Pull request by mohitahlawat2001**
## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [ ] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/security-quiz.ts [hunk 1]
```diff
@@ -627,6 +629,67 @@ const securityQuiz = [
...
+      "Wordlists are used to crack password hashes by trying out all possible passwords in the list. Wordlists are a core component of brute-force attacks.",
+    Link: "https://www.freecodecamp.org/news/the-power-of-wordlists-why-every-ethical-hacker-needs-one/"
+  },
+  {
+    Question: 
+      "How Rate Limiting helps in Cybersecurity?",
+      Answer: "It prevent brute force attacks, web scraping, and DDoS attacks.",
+      Distractor1: "It prevent SQL injection attacks.",
+      Distractor2: "It prevent Cross-site scripting attacks.",
+      Distraction3: "It prevent from phishing attacks.",
```
Comment (ilenia-magoni) [inline src/data/security-quiz.ts:649]: the error is for this line, you should fix the proeprty key

File: src/data/security-quiz.ts [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import Questions from "../components/Questions";
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:1]: this doesn't need to be here.
please remove

**Review by jdwilkin4 — COMMENTED** [review]
Please fix the formatting issues before we can continue with a review.

When you add new questions, the formatting should match what is already there.

**Comment by jdwilkin4** [comment]
I am closing this PR due to inactivity.

**Comment by mohitahlawat2001** [comment]
sorry for delay i`m working on another project

**Comment by mohitahlawat2001** [comment]
done make necessary changes

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**